packetbeat/sniffer/worker.go

```diff
@@ -15,7 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
```

Comment:
Maybe this file should be `decoders.go` now?

Reply:
Will do.